include/SDL3/SDL_hints.h

```diff
@@ -4128,6 +4128,20 @@ extern "C" {
  */
 #define SDL_HINT_WINDOWS_RAW_KEYBOARD "SDL_WINDOWS_RAW_KEYBOARD"
 
+/**
+ * A variable controlling whether to block hotkeys when raw keyboard events are enabled.
+ *
+ * The variable can be set to the following values:
+ *
+ * - "0": Hotkeys are not blocked. (default)
+ * - "1": Hotkeys are blocked.
+ *
+ * This hint can be set anytime.
+ *
+ * \since This hint is available since SDL 3.2.22.
+ */
+#define SDL_HINT_WINDOWS_RAW_KEYBOARD_NOHOTKEYS "SDL_WINDOWS_RAW_KEYBOARD_NOHOTKEYS"
```

Comment:
Can you invert this hint and its default to be simpler?
```suggestion
#define SDL_HINT_WINDOWS_RAW_KEYBOARD_HOTKEYS "SDL_WINDOWS_RAW_KEYBOARD_HOTKEYS"
```

Reply:
done